apps/web-giddh/src/app/subscription/subscription.component.html

```diff
@@ -1,353 +1,382 @@
+                                            (click)="deleteSession(element.sessionId, sessionIndex)"
+                                            *ngIf="userSessionId !== element.sessionId"
+                                        >
+                                            <i
+                                                class="icon-trash"
+                                                aria-hidden="true"
+                                                [matTooltip]="commonLocaleData?.app_delete"
+                                                [matTooltipPosition]="'above'"
+                                            ></i>
+                                        </button>
+                                    </td>
+                                </ng-container>
+
+                                <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+                                <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+                            </table>
+                        </div>
+                        <div class="no-data" *ngIf="!userSessionList?.length">
+                            <no-data
+                                [primaryMessage]="commonLocaleData?.app_no_entries_found"
+                                [secondaryMessage]="commonLocaleData?.app_search_suggestion"
+                                [secondaryMessageClass]="'p-0'"
+                            ></no-data>
+                        </div>
+                    </div>
+                </div>
+            </mat-tab>
+            <mat-tab [label]="localeData?.subscription?.tab_heading">
+                <ng-container
+                    class="wrap-tab-content-pd-15"
+                    *ngIf="selectedCompany?.planVersion === 1 && selectedCompany?.subscription?.status !== 'expired'"
+                >
+                    <old-subscription
+                        [localeData]="localeData"
+                        [commonLocaleData]="commonLocaleData"
+                        (isSubscriptionLoading)="isSubscriptionLoading = $event"
+                        (subscriptionId)="subscriptionId = $event"
+                    ></old-subscription>
+                </ng-container>
+                <ng-container *ngIf="selectedCompany?.planVersion === 2">
+                    <subscription-list></subscription-list>
+                </ng-container>
+            </mat-tab>
+        </mat-tab-group>
+
+        <!-- end of tab set -->
     </div>
-    <div class="input-container" [hidden]="!show">
-        <text-field
-            [type]="'text'"
-            [placeholder]="getSearchFieldText(fieldName)"
-            [cssClass]="'form-control search-table mat-field-border'"
-            [formControl]="formControl"
-            [autoFocus]="show"
-        ></text-field>
+
+    <div *ngIf="isCreateAndSwitchCompanyInProcess">
+        <giddh-page-loader></giddh-page-loader>
```

Comment:
remove extra div and appy condition in loader

Reply:
do not change anything on old subscription module